archean/egg-archean.json

```diff
@@ -0,0 +1,174 @@
+{
+    "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PANEL",
+    "meta": {
+        "version": "PTDL_v2",
+        "update_url": null
+    },
+    "exported_at": "2024-10-21T00:55:52+00:00",
+    "name": "Archean",
+    "author": "tueem@tomatentum.net",
+    "uuid": "01555bd3-155f-4ab6-b63d-c8bed0a3585f",
+    "description": "Archean invites you to unleash your creativity alone or with friends in a creative sandbox mode.Design and build vehicles, bases, rockets, space stations and much more using customizable modular blocks.Create unique configurations using components, program their behavior and explore an environment free of constraints.\\nTest your builds, refine your designs and defy the laws of physics.Archean is a game in development focused on engineering and build.",
+    "features": [],
+    "docker_images": {
+        "Ubuntu": "ghcr.io\/pelican-eggs\/yolks:ubuntu"
+    },
+    "file_denylist": [],
+    "startup": ".\/archean_start.sh",
+    "config": {
+        "files": "{}",
+        "startup": "{\n\"done\": \"Server started\"\n}",
+        "logs": "{}",
+        "stop": "^C"
+    },
+    "scripts": {
+        "installation": {
+            "script": "#!\/bin\/bash\n\nmv \/Archean-server-alpha \/mnt\/server\/Archean-server-alpha\n\napt update -y\napt install -y curl\ncurl -o \/mnt\/server\/archean_start.sh https:\/\/git.tomatentum.net\/tueem\/archean-server-pelican\/raw\/branch\/master\/archean_start.sh\nchmod -R 755 \/mnt\/server\/\n\necho ******************************************\necho Installation Completed\necho ******************************************",
```

Comment:
Is there any better or more "official" way to obtain their server files other than copying from their precompiled Docker image? If not, this should be fine, but not ideal in my opinion. Also, you should only `chmod` required files if possible. Assigning `755` to the whole home directory is bad practice.

Reply:
Unfortunately no, the only alternative would be to login use steamcmd and login using a account that has the game to download it, which I think is even worse.

Reply:
then use steamcmd with login. Where is the problem?

Reply:
I don't want to avoid steam but I think that users should not need to enter their private steam credentials everytime they want to create a server if there is a better way.

Reply:
> I don't want to avoid steam but I think that users should not need to enter their private steam credentials everytime they want to create a server if there is a better way.

Thats the way steam works. blame devs, why they want an auth.